reference/7.2/Microsoft.PowerShell.Core/About/about_Environment_Variables.md

```diff
@@ -359,7 +378,11 @@ The environment variables that store preferences include:
   - Use the `Set-ExecutionPolicy` cmdlet. Use the **Scope** parameter with
     a value of `Process`.
 
-    For more information, see [about_Execution_Policies][05].
+  - Manually set the environment variable. Changing the value of this variable
+    changes the execution policy current process.
```

Comment:
Thanks for the quick turnaround; I think "changes the execution policy current process." should be "changes the execution policy for the current process."